src/olympia/ratings/tests/test_admin.py

```diff
@@ -281,6 +281,32 @@ def test_search_by_ip(self):
             ['4.8.15.16', '4.8.15.16', '125.5.6.7', '125.1.2.3\n4.8.15.16 125.1.1.2']
         )
 
+    def test_can_delete_on_changelist_while_sorting_by_ip(self):
+        with core.override_remote_addr('125.1.1.2'):
+            rating2 = Rating.objects.create(
+                addon=self.addon,
+                user=user_factory(),
+                rating=5,
+                body='Lôrem Ipsûm',
+            )
+        user = user_factory(email='someone@mozilla.com')
+        self.grant_permission(user, 'Admin:Advanced')
+        self.grant_permission(user, 'Ratings:Moderate')
+        self.client.force_login(user)
+        data = {
+            '_selected_action': str(self.rating.pk),
+            'action': 'delete_selected',
+            # post=yes emulates the "Yes, I'm sure" on the delete confirmation.
+            'post': 'yes',
+        }
+        # o=5 is ordering by IP, that's what we want to test.
+        query_string = f'?addon={self.addon.pk}&o=5'
```

Comment:
Yeah, Django admin only references columns by their index unfortunately... We could find where `known_ip_adresses` column is on a model admin instance but even that is annoying, cause it could be dynamic... Or maybe I could grab it from the HTML of a response...

Reply:
The test above, `test_search_by_ip`, explicitly checks for the results with that same ordering. So I could do something similar, to make the test to fail and force us to update it if we ever change the columns ordering.

Reply:
https://github.com/mozilla/addons-server/pull/22245/commits/441067b6fbc1d8b1625482d965967545e96591c6